DOC-8619 Server 7.0β compatibility matrix

osfameron opened the pull request.

osfameron committed:
DOC-8619 Server 7.0β compatibility matrix

osfameron committed:
Correct server include

chvck commented on modules/project-docs/pages/compatibility.adoc:
```diff
@@ -93,9 +102,9 @@ The SDK follows http://semver.org/[Semantic Versioning^] and typically release a
+include::6.6@sdk:shared:partial$network-requirements.adoc[]
 
-NOTE: With Go 1.6, which does not have the Alternate Address feature, you need to use `?network=default` in the connection string for COuchbase Cloud.
+NOTE: With Go 1.6, which does not have the Alternate Address feature, you need to use `?network=default` in the connection string for Couchbase Cloud.
```
This isn't really related to this PR I guess but 1.6 *does* have the alternate address feature and doesn't require a query string param - even if it did it would need network=external anyway :D.

osfameron committed:
Remove invalid note

Charlie has confirmed with 1.6 against Cloud that this is unneeded:

	connStr := "couchbases://xxxxxx.dp.cloud.couchbase.com"
	cluster, err := gocb.Connect(connStr)
